src/About.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+	Content,
+	Container,
+	Section,
+	Heading,
+	Tile,
+	Notification,
+} from 'react-bulma-components';
+
+const About = () => {
+  return (
+		<Container>
+			<Section>
+				<Tile kind="ancestor" id="about">
+					<Tile kind="parent">
+						<Tile
+							kind="child"
+							renderAs={Notification}
+							color={"gray"}
+						>
+							<Heading weight="semibold" subtitle style={{ textAlign: 'center' }}>About Service</Heading>
+							<Content>
+								<p>Archive.NeoFS is a web application that allows users to create blockchain archives of any span (from block 0 to the current block or a custom range) directly in the browser. It operates fully client-side, leveraging streaming techniques to efficiently fetch and store blockchain data without requiring additional backend processing.</p>
```

Comment:
I tried to describe it well, but @roman-khimov do it better.

Reply:
> Archive.NeoFS is a web application that allows users to create blockchain archives of any span (from block 0 to the current block or a custom range) directly in the browser. It operates fully client-side, leveraging streaming techniques to efficiently fetch and store blockchain data without requiring additional backend processing.

Archive.NeoFS is a web application that allows users to create blockchain archives of any span (from block 0 to the current block or a custom range) directly in the browser. It operates fully client-side, leveraging standard NeoFS REST gateway APIs and in-browser streaming techniques to efficiently fetch and store blocks without requiring additional backend processing.